### PR TITLE
Add polling endpoint

### PR DESCRIPTION
**Pull request by ananthb86**
Fix swagger for polling endpoint
Add polling endpoint examples

### CODE REVIEW

File: vipps-checkout-api.md [hunk 1]
```diff
@@ -131,6 +126,13 @@ Here is an example integration written in JavaScript that will make a request to
           method: 'POST',
           headers: {
             'Content-Type': 'application/json',
+            'Vipps-System-Name': 'direct',
+            'Vipps-System-Version': '1.0',
+            'Vipps-System-Plugin-Name': 'direct',
+            'Vipps-System-Plugin-Version': '1.0',
+            'client_id': '<your client id>',
+            'client_secret': '<your client secret>',
+            'Ocp-Apim-Subscription-Key': '<your ocp apim subscription key>',
```
Comment (okpedersen) [inline vipps-checkout-api.md:135]: This si frontend code, where the merchant frontend calls the merchant backend. These headers should be handled by the backend.

File: docs/swagger.yaml [hunk 2]
```diff
@@ -49,8 +120,10 @@ paths:
           description: Successfully retrieved the CustomerData object for the given session id.
         '204':
           description: 'The specified session id is recognized, but no data is available yet (login flow not yet completed by user).'
```
Comment (okpedersen) [inline docs/swagger.yaml:122]: These responses no longer corresponds to what the system actually does, and should be updated.

File: docs/swagger.yaml [hunk 3]
```diff
@@ -33,12 +69,47 @@ paths:
       responses:
         '200':
           description: ''
-  /session/{sessionId}:
+  /v1/session/{sessionId}:
     get:
       tags:
         - Session
       summary: This endpoint is polled on by the checkout-frontend application while the user completes the login flow in another tab.
```
Comment (okpedersen) [inline docs/swagger.yaml:76]: This summary is not correct for the merchant polling endpoint.

**Review by okpedersen — CHANGES_REQUESTED** [review]

File: vipps-checkout-api.md (hunk 1 above)
Comment (ananthb86) [reply vipps-checkout-api.md:135]: Removed it here and added it to Request headers section

File: docs/swagger.yaml (hunk 2 above)
Comment (ananthb86) [reply docs/swagger.yaml]: Fixed it

**Review by okpedersen — APPROVED** [review]

**Pull request merged**